tests.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8.17-slim-bullseye
+
+WORKDIR /root/IoTuring
+
+COPY . .
+
+RUN pip install --no-cache-dir .[test]
+
+CMD python -m pytest
+
+# docker run --rm -it $(docker build -q -f tests.Dockerfile .)
```

Comment:
is this comment wanted ?

Reply:
It's the same comment as in DEVELOPMENT.md, you can build, run and than remove the container. It's just documentation inside the file, so you don't have to look it up somewhere else. Maybe I can add some clarification what is that.

Reply:
It's okay, I don't mind having it there